waterlog/src/Components/TankComponent.js

```diff
@@ -72,6 +72,7 @@ export class TankComponent extends Component {
 
         
```

Comment:
Just a question, are you trying to add spaces between the heading tags? I'd rather add margins or padding to the style for this

Reply:
...margins or padding depending on the background aesthetics of the objects. It just keeps the markup a bit cleaner.